Weekly W4 + management artifacts

GuillaumeDespaux opened the pull request.

GuillaumeDespaux committed:
Management + weekly w4

GuillaumeDespaux committed:
Update management_artifacts.md

GuillaumeDespaux committed:
Merge branch 'main' into management

Enonome reviewed (changes requested):
__In the management_artifacts:__
- [x] *(line 7)* You can change the 2nd "part" ("parts") in "sections".
- [x] *(line 7)* "bellow" takes only one 'L'
- [x] *(line 7)* replace the "to" of "access to" by "the"
- [x] *(line 152)* "contain" takes a 's'; "how the" -> "how they"; maybe "of" is more appropriated in replacement of "for".

__In the weekly_report_4:__
- [x] *(line 13)* not mandatory but "everyone" can be better than "everybody"

GuillaumeDespaux committed:
Fixing

Fix of :
https://github.com/algosup/2024-2025-project-1-fpga-team-2/pull/20#pullrequestreview-2381507990

Enonome reviewed (approved):
Ok for me

GuillaumeDespaux committed:
Merge branch 'main' into management

GuillaumeDespaux committed:
Update management_artifacts.md
